convert2rhel/utils.py

```diff
@@ -496,21 +496,35 @@ def download_pkg(
     if ret_code != 0:
         loggerinst.warning("Output from the yumdownloader call:\n%s" % (output))
 
-        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:
+        # add in the new envar that should be changed to along with the old one
+        allow_old_envar_names = ("CONVERT2RHEL_INCOMPLETE_ROLLBACK", "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK")
+
+        # search for either envar in the system to see if it has been set
+        if any(envvar in os.environ for envvar in allow_old_envar_names):
+
+            # check to see which envar was used
+            if "CONVERT2RHEL_INCOMPLETE_ROLLBACK" not in os.environ:
+                # give the warning about the old envar not being used anymore if it is uesed
+                loggerinst.warning(
+                    "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'"
+                    " environment variable.  Please switch to 'CONVERT2RHEL_INCOMPLETE_ROLLBACK'"
+                    " instead."
+                )
+
+                # give regular warning if the new envar is used
+                loggerinst.warning(
+                    "Couldn't download the %s package. This means we will not be able to do a"
+                    " complete rollback and may put the system in a broken state.\n"
+                    "'CONVERT2RHEL_INCOMPLETE_ROLLBACK' environment variable detected, continuing conversion." % (pkg)
+                )
+        else:
             loggerinst.critical(
                 "Couldn't download the %s package. This means we will not be able to do a"
                 " complete rollback and may put the system in a broken state.\n"
                 "Check to make sure that the %s repositories are enabled"
                 " and the package is updated to its latest version.\n"
                 "If you would rather ignore this check set the environment variable"
-                " 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'." % (pkg, system_info.name)
```

Comment:
One of the reasons to change away from using 'unsupported' is to also change the wording of the whole log message, not just the env var. You should lead the collaboration on how to word it here with the rest of the team but I can give some hints of why I don't think the message we have atm is the best

> Couldn't download the %s package. This means we will not be able to do a complete rollback and may put the system in a broken state.
Just not being able to download a package doesn't mean the system will be in a broken state (it can happen, but if you're a customer reading this they'll just be thinking of something like "git" etc.). Ideally we'd mention that we can't back up packages to do a proper rollback, and that this means we won't have a guarantee that the packages are restored in the event of a rollback. As well as a mention of how some types of packages not being restored can create a broken state

Reply:
Okay yeah, now I got the full scope of what I should be doing for this PR, as I though it was just an change of the names of the env var!. Thank you for clearing this up!

Reply:
That's part of it yeah! Rewriting to encompass all of these things can be tricky, can try to do it in our GChat with everyone or create like a GDoc with the phrases in and we can collaborate that way too. But it's EOY so either just set it up to be ready for after the new years now or change it up yourself for now and we can start later

Reply:
yeah that sounds like a great plan!  I will get this PR set up as much as I can. Then we all can take care of it next year! I will also go ahead and get a GDoc prepared for next year!